modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -44,49 +44,63 @@ Allocatable:
  pods:                        250
 ----
 
-. To set the max Pods per node on the worker nodes, create a YAML file that
-contains the kubelet configuration. For example, `max-worker-pods.yaml`:
+. To set the max Pods per node on the worker nodes, create a custom resource file
+that contains the kubelet configuration. For example,
+`worker-kube-config-change-maxPods-crd.yaml`:
 +
 ----
 apiVersion: machineconfiguration.openshift.io/v1
 kind: KubeletConfig
 metadata:
   name: set-max-pods
 spec:
-  machineConfigSelector: 01-worker-kubelet
+  machineConfigPoolSelector:
+    matchLabels:
+      custom-kubelet: large-pods
   kubeletConfig:
-    maxPods: 250
+    maxPods: 500
 ----
-
-. Apply the file by running:
++
+.. Run:
 +
 ----
-$ oc apply -f max-worker-pods.yaml
+$ oc label machineconfigpool worker custom-kubelet=large-pods
 ----
 
-. Verify the change by running:
+.. Run:
 +
 ----
-$ oc get kubeletconfigs set-max-pods -o yaml
+$ oc create -f worker-kube-config-change-maxPods-crd.yaml
 ----
 
-By default, only one machine is allowed to be unavailable when applying the kubelet-related configuration to the available worker nodes. For a large cluster, it can take a long time for the configuration change to be reflected. At any time, you can adjust the number of machines that are updating to speed up the process.
-
-.Procedure
-. Run:
+.. Run:
 +
 ----
-$ oc edit machineconfigpool worker
```

Comment:
@ahardin-rh can we keep the procedure to change the number of maxUnavailable nodes at a given time when rebooting the nodes to apply the kubelet config changes please?

Reply:
@ahardin-rh or perhaps put link to the procedure to change the number of maxUnavailable nodes at a given time when rebooting the nodes here.